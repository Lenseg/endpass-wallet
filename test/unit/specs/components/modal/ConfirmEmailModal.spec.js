import { shallow } from '@vue/test-utils';

import ConfirmEmailModal from '@/components/modal/ConfirmEmailModal';

describe('ConfirmEmailModal', () => {
  describe('render', () => {
    let wrapper;

    beforeEach(() => {
      wrapper = shallow(ConfirmEmailModal, {});
    });

    it('should be a Vue component', () => {
      expect(wrapper.isVueInstance()).toBeTruthy();
    });
  });
});
