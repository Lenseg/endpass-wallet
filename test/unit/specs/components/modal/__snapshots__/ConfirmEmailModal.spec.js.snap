// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmEmailModal render should match snapshot 1`] = `<!---->`;
